<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WayPointForm</class>
 <widget class="QDialog" name="WayPointForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WayPoint</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="textLabelName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelName_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Spot</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelName_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Country</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelLat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Latitude</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelAlt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Altitude</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelLon">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>73</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitude</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelUnitLat">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[°.min]</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelUnitLon">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[°.min]</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelUnitAlt">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[m]</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxAlt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>259</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>8000</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCountry">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>111</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSpot">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>61</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>11</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLat">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>161</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>9</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLon">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>211</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>52</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>79</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditSpot</tabstop>
  <tabstop>lineEditCountry</tabstop>
  <tabstop>lineEditLat</tabstop>
  <tabstop>lineEditLon</tabstop>
  <tabstop>spinBoxAlt</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>WayPointForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>WayPointForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditLat</sender>
   <signal>lostFocus()</signal>
   <receiver>WayPointForm</receiver>
   <slot>limitLatitude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditLon</sender>
   <signal>lostFocus()</signal>
   <receiver>WayPointForm</receiver>
   <slot>limitLongitude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
